common/src/main/java/com/flipkart/varadhi/Constants.java

```diff
@@ -4,6 +4,7 @@ public class Constants {
     public static final int RANDOM_PARTITION_KEY_LENGTH = 5;
     public static final String CONTEXT_KEY_BODY = "varadhi.body";
     public static final String CONTEXT_KEY_RESOURCE_HIERARCHY = "varadhi.resourceHierarchy";
+    public static final String USER_ID_HEADER = "x_user_id";
```

Comment:
add todo: this user is only for testing and x_ convention may cause it to be sent to the destination during consume